androidHackathonApp/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/MealsAdapter.kt

```diff
@@ -87,11 +87,10 @@ class MealsAdapter (var mealsList: List<Meal>) : RecyclerView.Adapter<MealsAdapt
     }
 
     fun getQuantityText(quantity: Int) : String {
```

Comment:
could we remove the space before the : please :)

Reply:
Could also just do 
```
fun getQuantityText(quantity: Int) = when (quantity) {
    0 -> "#    Reserved"
    ... etc
}
```

Reply:
we have literally just been saying this in slack :)

Reply:
I really love Kotlin, especially the way constructors work